From 2096f51cccdab7ebdc91524dafcbb15e217677d0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Thu, 2 Oct 2014 14:55:22 +0300
Subject: [PATCH 6/7] queue: Add missing break in switch

---
 plugins/elements/gstqueue.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/elements/gstqueue.c b/plugins/elements/gstqueue.c
index 35ad37f..f3e9297 100644
--- a/plugins/elements/gstqueue.c
+++ b/plugins/elements/gstqueue.c
@@ -694,6 +694,7 @@ gst_queue_locked_enqueue_event (GstQueue * queue, gpointer item)
       /* a new segment allows us to accept more buffers if we got EOS
        * from downstream */
       queue->unexpected = FALSE;
+      break;
     case GST_EVENT_GAP:
       apply_gap (queue, event, &queue->sink_segment, TRUE);
       break;
-- 
1.8.1.2

